### PR TITLE
Feature/rag function

### PR DESCRIPTION
**Pull request by akashAD98**

### CODE REVIEW

File: whisperplus/info_rag.txt
Comment (kadirnar) [inline whisperplus/info_rag.txt:1]: Do we need this file?

File: whisperplus/rag_main.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from pipelines.pipeline.chatbot import ChatWithVideo
```
[the remark this answers is not in the document]
Comment (akashAD98) [reply whisperplus/rag_main.py]: done with changes

File: whisperplus/info_rag.txt
Comment (akashAD98) [reply whisperplus/info_rag.txt:1]: i cant able to delet this so created this, can you delete this file from your end?

**Comment by kadirnar** [comment]
Thank you for your contribution ❤️

**Pull request merged**